src/vs/code/electron-sandbox/issue/issueReporterService.ts

```diff
@@ -1086,13 +1091,15 @@ export class IssueReporter extends Disposable {
 	private parseGitHubUrl(url: string): undefined | { repositoryName: string; owner: string } {
 		// Assumes a GitHub url to a particular repo, https://github.com/repositoryName/owner.
 		// Repository name and owner cannot contain '/'
-		const match = /^https?:\/\/github\.com\/([^\/]*)\/([^\/]*).*/.exec(url);
+		const match = /^https?:\/\/github\.com\/([^\/]*)\/([^\/]*)$/.exec(url);
```

Comment:
this is a tighter parse. 

this makes the assumption that the url we're checking is `https://github.com/repo-owner/repo` and nothing more. (in this case, we're basically grabbing the extension URL from the package.json, not the existing bugsUrl)

Reply:
Wouldn't it be better to fix `getExtensionGitHubUrl` so that if the url configured is a discussion URL, then that just returns `''` which seems to already be handled in this code?

Reply:
hmm yes that's true, i didn't see that function actually - will have a fix with that instead 😵‍💫